index.html

```diff
@@ -46,16 +47,164 @@ <h1>I’m Anthony <br> Glad to see you!</h1>
+                  <li>html</li>
+                  <li>css</li>
+                  <li>javascript</li>
+                  </ul>
+                  <a  class="viewpro-btn">See Project</a>
+                </div>
+
+                <div class="card">
+                  <img src="mobile-images/works/Portfolio3.png" alt="" class="img-center">
+                   <p class="project-title">Tonic</p>
+                   <div class="card-sub-head">
+                    <span class="canopy">CANOPY</span>
+                    <i class="fa fa-fw fa-circle"></i>
+                    <span class="sub-text"> Back End Dev</span>
+                    <i class="fa fa-fw fa-circle"></i>
+                    <span class="sub-text"> 2015</span>
+                  </div>
+                   <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul>
+                      <li>html</li>
+                      <li>css</li>
+                      <li>javascript</li>
+                      </ul>
+                      <a  class="viewpro-btn">See Project</a>
+                    </div>
+                    <div class="card">
+                      <img src="mobile-images/works/Portfolio2.png"  alt="" class="img-center">
+                       <p class="project-title">Multi-Post
+                        Stories</p>
+                        <div class="card-sub-head">
+                          <span class="canopy">CANOPY</span>
+                          <i class="fa fa-fw fa-circle"></i>
+                          <span class="sub-text"> Back End Dev</span>
+                          <i class="fa fa-fw fa-circle"></i>
+                          <span class="sub-text"> 2015</span>
+                        </div>
+                       <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                        <ul>
+                          <li>html</li>
+                          <li>css</li>
+                         
+                          <li>javascript</li>
+                          </ul>
+                          <a  class="viewpro-btn">See Project</a>
+                      </div>
+        </section>
+      <section class="aboutme">
+        <div class="header">
+          <h2 class="about-title">About <br> Myself</h2>
+          <p>
+            Hello I’m a software developer! I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+          <a class="lets-connect" href="#">LET’S CONNECT</a>
+          <ul class="social">
+            <li>
+              <a href="#">
+                <img src="mobile-images/Github.svg"  alt="Github">
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="mobile-images/Linkedin.svg" alt="Linkedin">
```

Comment:
Please make sure that all the icons are visible and not broken in the `About Myself` section. Kindly see the screenshot below.
![icons](https://user-images.githubusercontent.com/35504149/159133901-4b57c4fe-95fe-4fd6-a79c-cb12c1a1723a.png)

Reply:
Implemented 👍